SYCL/AtomicRef/max.cpp

```diff
@@ -1,7 +1,7 @@
 // RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %HOST_RUN_PLACEHOLDER %t.out
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
What is the reason for this reordering?

Reply:
This is the issue with how Git concatenates the two commits of this PR (the actual changes w/ deleted/moved files). It may be easier to review the actual changes by looking at https://github.com/intel/llvm-test-suite/pull/308/commits/f66a1229b0abd3e3caf47a462810efe0da16b307 specifically - the reordering would not be shown there.